source/guides/guides/network-requests.md

```diff
@@ -13,6 +13,9 @@ title: Network Requests
 - How to write declarative tests that resist flake
 {% endnote %}
 
+{% note info %}
+{% fa fa-cogs %} If your looking for a resource to make an HTTP request take a look at {% url "`cy.request()`" request %}
+{% endnote %}
```

Comment:
I would likely move this into the first header because it looks very easy to overlook next to the 'What you'll learn'

```suggestion
{% note info %}
**Note:** If your looking for a resource to make an HTTP request take a look at {% url "`cy.request()`" request %}
{% endnote %}
```